### ISSUE TITLE
[Bug] dashboard preset does not work anymore

### ISSUE THREAD

**Issue by jonormann** [dynatrace-oss/terraform-provider-dynatrace#71]
Hi,
when sharing a dashboard as a preset I got an error. 
```
│ Error: {
│   "code": 400,
│   "message": "Bad request attempting to modify preset state when not allowed to",
│   "constraintViolations": null
│ }
```
Dashboard is created witht the same Terraform script. Sharing to groups/ users works.
TF Code:
```
resource "dynatrace_dashboard_sharing" "my_dashboard_sharing" {
  enabled = true
  dashboard_id = dynatrace_dashboard.my_dashboard.id
  preset = true
  permissions {
    permission {
      type = "ALL"
      level = "VIEW"
    }
  }
}
```
Best
Joerg

**Comment by Dynatrace-Reinhard-Pilz** [comment]
Hello Joerg,

You've unfortunately stumbled across a regression bug within the Dynatrace REST API.
The issue is supposed to be fixed with Cluster version 1.226.

Sorry for the inconvenience,
Reinhard

**Comment by jonormann** [comment]
okay, good to know! Thanks!
Best
Joerg
